wdl2cwl/tests/oop_cwl_files/bowtie_1.cwl

```diff
@@ -0,0 +1,60 @@
+class: CommandLineTool
+id: Bowtie
```

Comment:
Does this file match the `main.py` output?

Reply:
Yes. But the requirements is incomplete.

Am still exploring the command section from the WDL file.